fed/mosaic-cell/src/main/resources/CRegionScheme.json

```diff
@@ -1,180 +1,281 @@
+                    "description": "The downlink supports two individual paths: Point-to-point communication (unicast) and Point-to-multipoint communication (multicast).",
+                    "$ref": "#/definitions/mobileNetworkProperties.downlink"
+                }
+            },
+            "anyOf": [
+                { "required": [ "area", "id", "uplink", "downlink" ] },
+                { "required": [ "polygon", "id", "uplink", "downlink" ] }
+            ]
+        },
+        "mobileNetworkProperties.uplink": {
+            "title": "mobileNetworkProperties.uplink",
+            "description": "Object to define the uplink of a network. The uplink direction only allows point-to-point communication (unicast). It is composed of the three nested models for delay, transmission configuration and capacity.",
+            "type": "object",
+            "properties": {
+                "delay": {
+                    "description": "The delay of the uplink.",
+                    "$ref": "#/definitions/delay"
+                },
+                "transmission": {
+                    "description": "The transmission model of the uplink.",
+                    "$ref": "#/definitions/transmission"
+                },
+                "capacity": {
+                    "description": "The capacity of the uplink.",
                     "anyOf": [
-                        { "required": [ "area", "id", "uplink", "downlink" ] },
-                        { "required": [ "polygon", "id", "uplink", "downlink" ] }
+                        { "type": "integer", "minimum": 0 },
+                        { "type": "string", "pattern": "^unlimited$"}
                     ]
                 }
-            ]
-        }
-    },
-    "definitions": {
+            },
+            "required": [ "delay", "transmission", "capacity" ] 
+
+        },
+        "mobileNetworkProperties.downlink": {
+            "title": "mobileNetworkProperties.downlink",
+            "description": "Object to define the downlink of a network. The downlink supports two individual paths: Point-to-point communication (unicast) and Point-to-multipoint communication (multicast).",
+            "type": "object",
+            "properties": {
+                "unicast": {
+                    "description": "The properties of the unicast communication.",
+                    "$ref": "#/definitions/unicast"
+                },
+                "multicast": {
+                    "description": "The properties of the multicast communication.",
+                    "$ref": "#/definitions/multicast"
+                },
+                "capacity": {
+                    "description": "The capacity of the downlink.",
+                    "anyOf": [
+                        { "type": "integer", "minimum": 0 },
+                        { "type": "string", "pattern": "^unlimited$"}
+                    ]
+                }
+            },
+            "required": [ "unicast", "multicast", "capacity" ]
```

Comment:
As I understood, the jsconscheme2md script cannot interpret these definitions of 'required'. Is that true? If so, please add a required field to the corresponding properties. This should be done for the whole file.